ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/route_maps.py

```diff
@@ -119,6 +109,22 @@ def _route_maps_vrf_default(self) -> dict | None:
                 "match": ["ip address prefix-list PL-STATIC-VRF-DEFAULT"],
             }
 
+        if self._underlay_filter_redistribute_connected:
```

Comment:
Need to rebase to take into account the new route-map syntax - maybe let's keep the ordering of generation (RM-CONN before RM-EVPN if both present?)